browser/UHR.js

```diff
@@ -109,7 +109,10 @@ UHR.prototype._doRequest = function (parameters) {
 			requestError = new Error(xhr.statusText || ERROR_CONNECTION);
 			reject(requestError);
 		};
-		xhr.onloadend = function () {
+		xhr.onreadystatechange = function () {
+			if (xhr.readyState !== 4) {
+				return;
+			}
```

Comment:
Nono, it's exactly READYState. Same as loadend event for real browsers. And it readyState don't have any error codes.